<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>PrinterSettingsDialog</class>
  <widget class="QDialog" name="PrinterSettingsDialog">
    <property name="styleSheet">
      <string notr="true">
    background-color: rgb(42, 45, 46);
    color: white; 
      </string>
    </property>
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>379</width>
        <height>337</height>
      </rect>
    </property>
    <property name="windowTitle">
      <string>Настройки</string>
    </property>
    <property name="windowIcon">
      <iconset>
        <normaloff>:/img/logo-1.png</normaloff>
      </iconset>
    </property>
    <widget class="QDoubleSpinBox" name="input_tarif">
      <property name="geometry">
        <rect>
          <x>280</x>
          <y>90</y>
          <width>71</width>
          <height>25</height>
        </rect>
      </property>
      <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
        </sizepolicy>
      </property>
      <property name="font">
        <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
          <stylestrategy>PreferAntialias</stylestrategy>
        </font>
      </property>
      <property name="styleSheet">
        <string notr="true">background-color: rgb(77, 77, 77);</string>
      </property>
      <property name="minimum">
        <double>0.000000000000000</double>
      </property>
      <property name="maximum">
        <double>10000.000000000000000</double>
      </property>
    </widget>
    <widget class="QLabel" name="Tarif">
      <property name="geometry">
        <rect>
          <x>20</x>
          <y>90</y>
          <width>221</width>
          <height>25</height>
        </rect>
      </property>
      <property name="font">
        <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
        </font>
      </property>
      <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
        <string>Тариф электроэнергии, кВТ/ч</string>
      </property>
    </widget>
    <widget class="QDoubleSpinBox" name="input_q_trash">
      <property name="geometry">
        <rect>
          <x>280</x>
          <y>130</y>
          <width>71</width>
          <height>25</height>
        </rect>
      </property>
      <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
        </sizepolicy>
      </property>
      <property name="font">
        <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
          <stylestrategy>PreferAntialias</stylestrategy>
        </font>
      </property>
      <property name="styleSheet">
        <string notr="true">background-color: rgb(77, 77, 77);</string>
      </property>
      <property name="minimum">
        <double>0.000000000000000</double>
      </property>
      <property name="maximum">
        <double>10000.000000000000000</double>
      </property>
    </widget>
    <widget class="QLabel" name="Q_trash">
      <property name="geometry">
        <rect>
          <x>20</x>
          <y>130</y>
          <width>201</width>
          <height>25</height>
        </rect>
      </property>
      <property name="font">
        <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
        </font>
      </property>
      <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
        <string>Коэффициент выбраковки</string>
      </property>
    </widget>
    <widget class="QDoubleSpinBox" name="input_price_pastik">
      <property name="geometry">
        <rect>
          <x>280</x>
          <y>170</y>
          <width>71</width>
          <height>25</height>
        </rect>
      </property>
      <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
        </sizepolicy>
      </property>
      <property name="font">
        <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
          <stylestrategy>PreferAntialias</stylestrategy>
        </font>
      </property>
      <property name="styleSheet">
        <string notr="true">background-color: rgb(77, 77, 77);</string>
      </property>
      <property name="minimum">
        <double>0.000000000000000</double>
      </property>
      <property name="maximum">
        <double>10000.000000000000000</double>
      </property>
    </widget>
    <widget class="QLabel" name="Price_plastik">
      <property name="geometry">
        <rect>
          <x>20</x>
          <y>170</y>
          <width>181</width>
          <height>25</height>
        </rect>
      </property>
      <property name="font">
        <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
        </font>
      </property>
      <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
        <string>Стоимость катушки, руб</string>
      </property>
    </widget>
    <widget class="QDoubleSpinBox" name="input_overprice">
      <property name="geometry">
        <rect>
          <x>280</x>
          <y>210</y>
          <width>71</width>
          <height>25</height>
        </rect>
      </property>
      <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
        </sizepolicy>
      </property>
      <property name="font">
        <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
          <stylestrategy>PreferAntialias</stylestrategy>
        </font>
      </property>
      <property name="styleSheet">
        <string notr="true">background-color: rgb(77, 77, 77);</string>
      </property>
      <property name="minimum">
        <double>0.000000000000000</double>
      </property>
      <property name="maximum">
        <double>10000.000000000000000</double>
      </property>
    </widget>
    <widget class="QLabel" name="Over_price">
      <property name="geometry">
        <rect>
          <x>20</x>
          <y>210</y>
          <width>181</width>
          <height>25</height>
        </rect>
      </property>
      <property name="font">
        <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
        </font>
      </property>
      <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
        <string>Наценка, %</string>
      </property>
    </widget>
    <widget class="QLabel" name="weight_plastik">
      <property name="geometry">
        <rect>
          <x>20</x>
          <y>250</y>
          <width>181</width>
          <height>25</height>
        </rect>
      </property>
      <property name="font">
        <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
        </font>
      </property>
      <property name="styleSheet">
        <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
        <string>Вес катушки, гр</string>
      </property>
    </widget>
    <widget class="QSpinBox" name="plastik_menu">
      <property name="geometry">
        <rect>
          <x>280</x>
          <y>250</y>
          <width>71</width>
          <height>25</height>
        </rect>
      </property>
      <property name="styleSheet">
        <string notr="true">background-color: rgb(77, 77, 77);</string>
      </property>
      <property name="minimum">
        <number>0</number>
      </property>
      <property name="maximum">
        <number>10000</number>
      </property>
      <property name="value">
        <number>0</number>
      </property>
      <property name="font">
        <font>
          <pointsize>10</pointsize>
          <bold>true</bold>
          <stylestrategy>PreferAntialias</stylestrategy>
        </font>
      </property>
    </widget>
    <widget class="QLabel" name="warning_text">
      <property name="geometry">
        <rect>
          <x>30</x>
          <y>10</y>
          <width>311</width>
          <height>51</height>
        </rect>
      </property>
      <property name="styleSheet">
        <string notr="true">color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
        <string>ВНИМАНИЕ! Перед установкой параметра &quot;Коэффициент выбраковки&quot; прочитайте раздел Помощь, с описанием формулы расчета!</string>
      </property>
      <property name="alignment">
        <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
        <bool>true</bool>
      </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
        <rect>
          <x>20</x>
          <y>300</y>
          <width>331</width>
          <height>27</height>
        </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
          <widget class="QPushButton" name="save">
            <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 80, 0);</string>
            </property>
            <property name="text">
              <string>Сохранить</string>
            </property>
          </widget>
        </item>
        <item>
          <widget class="QPushButton" name="exit">
            <property name="styleSheet">
              <string notr="true">background-color: rgb(0, 80, 0);</string>
            </property>
            <property name="text">
              <string>Отмена</string>
            </property>
          </widget>
        </item>
      </layout>
    </widget>
  </widget>
  <resources/>
  <connections/>
</ui>